usaspending_api/search/tests/integration/spending_by_category/test_spending_by_recipient.py

```diff
@@ -386,3 +382,56 @@ def test_recipient_search_text_duns(client, monkeypatch, elasticsearch_transacti
     }
     assert resp.status_code == status.HTTP_200_OK, "Failed to return 200 Response"
     assert resp.json() == expected_response
+
+
+def test_single_level_in_recipient(client, monkeypatch, elasticsearch_award_index, awards_and_transactions):
+    setup_elasticsearch_test(monkeypatch, elasticsearch_award_index)
+
+    resp = client.post(
+        "/api/v2/search/spending_by_category/recipient",
+        content_type="application/json",
+        data=json.dumps(
+            {
+                "filters": {"time_period": [{"start_date": "2024-09-28", "end_date": "2025-09-28"}]},
+                "category": "recipient",
+                "spending_level": "awards",
+                "page": 1,
+            }
+        ),
+    )
+    expected_response = {
+        "category": "recipient",
+        "spending_level": "awards",
+        "limit": 10,
+        "page_metadata": {"page": 1, "next": None, "previous": None, "hasNext": False, "hasPrevious": False},
+        "results": [
+            {
+                "amount": 0.0,
+                "recipient_id": "26b30496-92ab-426a-ac30-c75803307041-C",
+                "name": "C RECIPIENT",
+                "code": "DUNS009",
+                "uei": "UEI009",
+                "total_outlays": 0.0,
+            },
+            {
+                "amount": 0.0,
+                "recipient_id": "8301261f-8fd1-49d1-9720-8afa3ce6e26f-P",
+                "name": "P RECIPIENT",
+                "code": "DUNS010",
+                "uei": "UEI010",
+                "total_outlays": 0.0,
+            },
+            {
+                "amount": 0.0,
```

Comment:
This is actually a good point brought up by the test case. Can you add logic that sets the return of `reicpient_id` to `None` if a level isn't found? This should only occur for our special cases such as `MULTIPLE RECIPIENTS`. Additionally, this allows the frontend to carry forward their logic that greys out the recipient's name (avoiding a link to recipient page) when the ID isn't found.